lib/index.js

```diff
@@ -64,6 +64,12 @@ function Client(options) {
   }
 }
 
+Client.prototype.createBucket = function createBucket(bucket, cb) {
+  this.s3.createBucket({
+    Bucket: bucket
+  }, cb);
+}
+
```

Comment:
Yep, will do that if and when we actually manage to get CI working with S3Ninja. Which I am still not 100% convinced will be a thing.

Also I hacked this in and have no idea if it matches the expected format for a new API function.